turbopack/crates/turbopack-node/src/transforms/webpack.rs

```diff
@@ -472,29 +473,60 @@ impl EvaluateContext for WebpackLoaderContext {
         pool: &NodeJsPool,
     ) -> Result<()> {
         match data {
-            InfoMessage::FileDependency { path } => {
-                // TODO We might miss some changes that happened during execution
-                // Read dependencies to make them a dependencies of this task. This task will
-                // execute again when they change.
-                self.cwd.join(path).read().await?;
-            }
-            InfoMessage::BuildDependency { path } => {
-                // TODO We might miss some changes that happened during execution
-                BuildDependencyIssue {
-                    context_ident: self.context_ident_for_issue,
-                    path: self.cwd.join(path).to_resolved().await?,
+            InfoMessage::Dependencies {
+                env_variables,
+                file_paths,
+                directories,
+                build,
+            } => {
+                // Track dependencies of the loader task
+                // TODO:Because these are reporter _after_ the loader actually read the dependency
+                // there is a race condition where we may miss updates that race
+                // with the loader execution.
+
+                // Track all the subscriptions in parallel, since certain loaders like tailwind
```

Comment:
Not at all sure if this is the best way to do this

Reply:
If you're concerned that this just causes an overload because of 40000 concurrent reads: that could also occur in Turbopack itself when exploring the module graph in parallel, so there are some semaphores to limit that already.